pingsrv.go

```diff
@@ -163,3 +164,14 @@ func grpcClient() {
 		pingClientCall(dest, count, *payloadFlag)
 	}
 }
+
+func setGRPCDestination(dest string) string {
```

Comment:
let's change this to GRPCDestination(dest string, port string)  // ie a pure function with no global access (it doesn't set anything and doesn't need to read anything)

that way you can add a test for it too

Reply:
OK